scripts/ci-integration.sh

```diff
@@ -23,8 +23,8 @@ REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 source "${REPO_ROOT}/hack/ensure-go.sh"
 
 MAKE="make"
-KIND_VERSION="v0.5.0"
-KUBECTL_VERSION="v1.15.3"
+KIND_VERSION="v0.6.1"
```

Comment:
Any reason not to use v0.7.0 here?

Reply:
@chuckha I agree there, but we can also override the k8s version used by kind as well. That is something we can track in a separate issue, though.